sw/device/mask_rom/mask_rom_test_base.c

```diff
@@ -0,0 +1,52 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "sw/device/lib/base/stdasm.h"
+#include "sw/device/lib/pinmux.h"
+#include "sw/device/lib/runtime/hart.h"
+#include "sw/device/mask_rom/mask_rom.h"
+#include "sw/device/rom_exts/rom_ext_manifest_parser.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+// This is the expected ROM_EXT Manifest Identifier
+static const uint32_t kRomExtIdentifierExpected = 0x4552544F;
+
+typedef void(boot_fn)(void);
```

Comment:
This typedef is only used twice. Do we *really* need it?

Reply:
This file has been deleted.

Done.